src/module/item/physical/document.ts

```diff
@@ -423,6 +427,28 @@ abstract class PhysicalItemPF2e<TParent extends ActorPF2e | null = ActorPF2e | n
         }
     }
 
+    override getEmbeddedDocument(
+        embeddedName: string,
+        id: string,
+        { strict }: { strict: true },
+    ): foundry.abstract.Document;
+    override getEmbeddedDocument(
+        embeddedName: string,
+        id: string,
+        { strict }: { strict: false },
+    ): foundry.abstract.Document | undefined;
+    override getEmbeddedDocument(
```

Comment:
This needs the third signature overload

Reply:
I tried adding it just now but it error'd, so I'm going to set the argument to options instead

![image](https://github.com/foundryvtt/pf2e/assets/1286721/6884ff60-e6df-4810-99e5-ecd158f2b427)
![image](https://github.com/foundryvtt/pf2e/assets/1286721/ccfc6ea6-b9e0-4ba2-b5bc-bd75c9b1687c)